src/addRemove.js

```diff
@@ -0,0 +1,103 @@
+import completed, { getTaskLS } from './interactive.js';
+
+export function myList(task, completed = false, index, list) {
+  const todo = { task, completed, index };
+  list.push(todo);
+  return list;
+}
+
+export default function addtodoItem(e) {
+  e.preventDefault();
+  const todoInput = document.getElementById('addlist').value;
+  const toDoObjects = getTaskLS();
+  if (todoInput === '') {
+    return;
+  }
+  myList(todoInput, false, toDoObjects.length, toDoObjects);
+  localStorage.setItem('todos', JSON.stringify(toDoObjects));
+
+  window.location.reload();
+  document.getElementById('addlist').value = '';
+}
+
+function arrangeIndex(toDoObjects) {
+  toDoObjects.forEach((todo, i) => {
+    todo.index = i;
+  });
+  localStorage.setItem('todos', JSON.stringify(toDoObjects));
+}
+
+function removeLocalTodos(e) {
+  const task = e.target;
+  const toDoObjects = getTaskLS();
+
+  let todoIndex = task.parentElement.parentElement.parentElement.id;
+  todoIndex = +todoIndex; // Converting to number
+  toDoObjects.forEach((element, x) => {
+    if (element.index === todoIndex) {
+      toDoObjects.splice(x, 1);
+      localStorage.setItem('todos', JSON.stringify(toDoObjects));
+    }
+  });
+  arrangeIndex(toDoObjects);
+  window.location.reload();
+}
+
+export function clearCompleted(e) {
+  e.preventDefault();
+  let toDoObjects = getTaskLS();
```

Comment:
As you are using ES6, It will be better to use arrow function instead of normal function